docs/source/quasi_dags.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>Mention something about parallel trends assumption?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Good idea. I've added an admonition box about parallel trends and also added it as a glossary term.</p>